src/main/java/com/google/devtools/build/lib/rules/cpp/CppActionConfigs.java

```diff
@@ -938,24 +938,26 @@ public static String getCppActionConfigs(
                 "      flag: '-o'",
                 "      flag: '%{output_file}'",
                 "    }",
-                "    flag_group {",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.quote_paths'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.bracket_paths'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.system_paths'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.cpp_defines'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.cpp_includes'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.cl_args'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.lipo_info'",
-                "      flag: '-R'",
-                "      flag: '.gnu.switches.text.annotation'",
-                "    }",
+                ifTrue(
```

Comment:
Ah I missed this one, can you pls use ifMac there?

Reply:
`ifMac` is the opposite of what we want. These lines should only be added on non-MacOS platforms.

Reply:
Then use ifLinux pls, windows crosstool is not patched by this code.

Reply:
I believe this should also be applied to FreeBSD.

Reply:
That is covered by 'linux' cppPlatform :) https://github.com/bazelbuild/bazel/blob/275ae45b1228bdd0f912c4fbd634b29ba4180383/src/main/java/com/google/devtools/build/lib/rules/cpp/CppToolchainInfo.java#L353 I know, this is not very robust :)

Reply:
OK, changed to use `ifLinux`. I also added a comment that FreeBSD counts as `LINUX`.